src/hotrod/impl/protocol/Codec20.cpp

```diff
@@ -275,6 +275,11 @@ void Codec20::writeClientListenerParams(transport::Transport& t, const ClientLis
     this->writeNamedFactory(t, clientListener.converterFactoryName, converterFactoryParams);
 }
 
+void Codec20::writeClientListenerInterests(transport::Transport& t, unsigned char) const
+{
+	// no op until 2.6 protocol
```

Comment:
I guess no one will invoke it at this version, but maybe throw an exception ?

Reply:
No method is used by the add listener operation and needs to be transparent for protocol version lower than 2.6